### DISCUSSION TITLE
ROB_200-070-0 multisensor

### DISCUSSION THREAD

**Discussion by TP51** [Koenkk/zigbee2mqtt#17445] [Devices]
Can anyone tell me how to integrate the 'ROB_200-070-0'  4-in-1 multi sensor in zigbee2mqtt? I know that there are several ROB items integrated already but not this one. I would like to know what/how to do.

**Comment by jhaand** [comment]
This PR will add support for this multisensor. 
https://github.com/Koenkk/zigbee-herdsman-converters/pull/7876
